prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -9,24 +9,10 @@ import java.io.FilenameFilter
 import java.io.InputStream
 
 /**
- * The metadata object is a singleton representing all metadata loaded for MappableTables
+ * A metadata object contains all the metadata including schemas, tables, valuesets, and organizations.
  */
-object Metadata {
-    private const val schemaExtension = ".schema"
-    private const val valueSetExtension = ".valuesets"
-    private const val tableExtension = ".csv"
-    private const val defaultMetadataDirectory = "./metadata"
-    private const val schemasSubdirectory = "schemas"
-    private const val valuesetsSubdirectory = "valuesets"
-    private const val tableSubdirectory = "tables"
-
-    private val PRIME_ENVIRONMENT = System.getenv("PRIME_ENVIRONMENT") ?: ""
-
-    private val ext = if (PRIME_ENVIRONMENT.isNotEmpty()) "-" + PRIME_ENVIRONMENT else PRIME_ENVIRONMENT
-
-    private val organizationsList = "organizations$ext.yml"
-
-    private var schemas = mapOf<String, Schema>()
+class Metadata {
+    private var schemaStore = mapOf<String, Schema>()
```

Comment:
Going from a global to a passed in value allows for dependency injection. It's a big change to our code in lines affected, but not so much from a conceptual point of view.